tests/test_phenogenon.py

```diff
@@ -6,7 +6,7 @@
 import json
 import gzip
 import numpy as np
-sys.path.append('lib')
+sys.path.append('phenogenon')
```

Comment:
Is this line needed?

Reply:
No, it isn't as I added `from phenogenon import...` in a later commit

(it wasn't working with just a `sys.path.append`) - do you have any idea why?

Reply:
No idea, but I don't like `sys.path.append` anyway :) Shall we remove this line then?

Reply:
Sure - have just pushed this...